src/types/basic.ts

```diff
@@ -10,6 +10,7 @@ export type SdkConfig = {
   trackPages: boolean
```

Comment:
We could rename this file to `base` instead

Reply:
We can. So we can agree with `base`